### PR TITLE
feat: complete deprecation of Unscaled

### PR DESCRIPTION
**Pull request by nstarman**
Follows #14417 

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: docs/changes/units/16707.api.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The deprecation for ``units.format.Unscaled`` has been completed and the class has been
```
Comment (pllim) [inline docs/changes/units/16707.api.rst:1]: Can be simplified.
```suggestion
The deprecated ``units.format.Unscaled`` has been
```

**Comment by pllim** [comment]
Maybe tests have to be updated too? CI is failing bad.

**Review by mhvk — APPROVED** [review]
Looks good, but suggestion from @pllim probably good. Not sure it is worth the CI, so will approve and let @pllim decide that.

**Comment by pllim** [comment]
> Not sure it is worth the CI

Change log, no. But tests need fixing, so CI needs to rerun anyway at some point.

**Comment by pllim** [comment]
Thanks!

**Pull request merged**